lib/session/messageSession.ts

```diff
@@ -61,7 +61,6 @@ export interface MessageSessionOptionsBase {
    * @property {number} [maxMessageWaitTimeoutInSeconds] The maximum amount of idle time the session
    * receiver will wait ater a message has been received. If no messages are received in that
    * time frame then the session will be closed.
-   * - **Default**: `60` seconds.
    */
   maxMessageWaitTimeoutInSeconds?: number;
```

Comment:
Penning the ideas that we agreed upon:
- MessageSession.maxMessageWaitTimeoutInSeconds is set to whatever value provided by the user while accepting a MessageSession. By default its value is undefined.
- It should be explicitly set to `2 seconds` while receiving messages in a batch.
- It should be set explicitly to `60 seconds` while calling MessageSession.receive() from the SessionManager.

Reply:
Currently, in both cases, on timeout, the MessageSession is being closed. But we do not want that to happen when a user is using `receive` or `receiveBatch` after accepting a session.
We only want to close the MessageSession on timeout when using the SessionManager.

Reply:
Also, why do we want to have this as an option on the MessageSession?
I would expect receiveBatch() to have it as a third parameter just like we do in the batchingReceiver. 
And receive() can have it in its options bag.

Reply:
both the points have been addressed in the latest commit.